### PR TITLE
[@kadena/graph] , [@kadena/graph-client] Non-nullable edges & transfers and pagination related UI adjustments 

### PR DESCRIPTION
**Pull request by nil-amrutlal-dept**
- Pagination Component was implemented 
- Implemented items per page in Transfers 
- Removed unused logger on `graph`. Might be nice to implement a uniform solution within the `graph` project 
- The transfers on each chain show the counterpart in the correct order. That means transfers going to chain 0 -> chain 1, show as chain 0 -> chain 1 even if we are looking at chain 1 transfers (see image below). For these changes, some additional functions were developed and the overall logic was refined

image

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205916745201980

### CODE REVIEW

File: packages/apps/graph-client/src/components/compact-transfers-table/compact-transfers-table.tsx [hunk 1]
```diff
@@ -56,72 +81,76 @@ export const CompactTransfersTable = (
...
-                    href={`${routes.TRANSACTIONS}/${edge?.node.requestKey}`}
-                  >
-                    <span title={edge?.node.requestKey}>
-                      {truncate(edge?.node.requestKey)}
+                  <Link href={`${routes.TRANSACTIONS}/${transfer.requestKey}`}>
+                    <span title={transfer.requestKey}>
+                      {truncate(transfer.requestKey)}
                     </span>
                   </Link>
                   /
```
Comment (MRVDH) [inline packages/apps/graph-client/src/components/compact-transfers-table/compact-transfers-table.tsx:147]: only add the / if there is a crossChainCounterPart

File: packages/apps/graph-client/src/components/extended-transfers-table/extended-transfers-table.tsx [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+                    </Link>
+                  ) : (
+                    <span style={{ color: 'lightgray' }}>N/A</span>
+                  )}
+                </Table.Td>
+                <Table.Td>
+                  <Link href={`${routes.TRANSACTIONS}/${edge.node.requestKey}`}>
+                    {edge.node.requestKey}
+                  </Link>
+                  /
```
Comment (MRVDH) [inline packages/apps/graph-client/src/components/extended-transfers-table/extended-transfers-table.tsx:234]: only add the / if there is a edge.node.crossChainTransfer

**Review by MRVDH — APPROVED** [review]

File: packages/apps/graph-client/src/components/compact-transfers-table/compact-transfers-table.tsx (hunk 1 above)
Comment (nil-amrutlal-dept) [reply packages/apps/graph-client/src/components/compact-transfers-table/compact-transfers-table.tsx]: Fixed

File: packages/apps/graph-client/src/components/extended-transfers-table/extended-transfers-table.tsx (hunk 2 above)
Comment (nil-amrutlal-dept) [reply packages/apps/graph-client/src/components/extended-transfers-table/extended-transfers-table.tsx]: Fixed

**Pull request merged**